log.lua

```diff
@@ -0,0 +1,10 @@
+--
```

Comment:
I think creating this file is a bit overkill.. I would move this function to config.. or having a more generic `util` module.. what do you think?

Reply:
I agree it's a bit off. I wanted to get rid of the duplication, and thought we could eventually add levels and stuff; but indeed we may not even need levels any time soon, so, I'm ok with some other option. What else would this `util` include besides the logs? I could just drop this commit also.